transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -378,10 +380,11 @@ protected void finishModelStage(final OperationContext context, final ModelNode
                     ModelNode resourceModel = model.getModel();
                     if (resourceModel.hasDefined(mutualAttributeName) && resourceModel.get(mutualAttributeName).asBoolean()) {
                         resourceModel.get(mutualAttributeName).set(new ModelNode(false));
+                        context.addResponseWarning(Level.WARNING, "Please note that only journal store or jdbc store can be used, thus '" + mutualAttributeName + "' has been disabled.");
```

Comment:
@rhusar I am actually looking for feedback here - is it the right way to fix it ? is the message good enough ? If the only issue is i18n, I'm happy to work on it :)